client/lib/promote-post/index.ts

```diff
@@ -86,7 +86,7 @@ export async function loadDSPWidgetJS(): Promise< void > {
 const shouldHideGoToCampaignButton = () => {
 	// App versions higher or equal than 22.9.rc-1 should hide the button
 	const deviceInfo = getMobileDeviceInfo();
-	return versionCompare( deviceInfo?.version, '22.9.rc-1', '>=' );
+	return versionCompare( deviceInfo?.version, '22.9-rc-1', '>=' );
```

Comment:
Just to double-check, what version should we put here? The comment says 22.9.rc-1, but the code says 22.9-rc-1

Reply:
You are right, I forgot to change the comments. The correct version should be 22.9-rc-1.
Let me update the comment.

More context about the version here: https://github.com/Automattic/wp-calypso/pull/79966#issuecomment-1662011776